src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,121 @@
 package core.basesyntax;
 
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int SCALE = 2;
+    private int capacity;
+    private int size;
+    private int threshold;
```

Comment:
capacity and threshold are useless field, please remove this, this will simplify the code. We can obtain capacity based on .length of buckets, and threshold is just a product of capacity and load factor